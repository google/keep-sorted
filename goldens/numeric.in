Numeric sorting:
// keep-sorted-test start numeric=yes
FOO_100
FOO_2
FOO_3
BAR_1
BAR_2
BAR_10
BAR_99999999999999999999999999999999999999999999999
// keep-sorted-test end


Mixed numeric and preference sorting with sticky comments:
deployment_state = [
  // keep-sorted-test start numeric=yes prefix_order=INIT,ROLLOUT,COMPLETE
  // All done.
  COMPLETE,
  // Start initialisation
  INIT_1,
  INIT_5,
  INIT_10,
  // Only deploy to 0.1%
  ROLLOUT_0_1,
  // just one percent.
  ROLLOUT_1,
  // Nearly done...
  ROLLOUT_100,
  ROLLOUT_10,
  ROLLOUT_5,
  ROLLOUT_50,
  // keep-sorted-test end
]

Prefix containing a number with numeric sorting:
droid_components = [
  // keep-sorted-test start numeric=yes prefix_order=R2,C3
  C3PO_HEAD,
  C3PO_ARM_L,
  R4_MOTIVATOR,
  C3PO_ARM_R,
  R2D2_BOLTS_10_MM,
  R2D2_PROJECTOR,
  R2D2_BOLTS_5_MM,
  // keep-sorted-test end
]
