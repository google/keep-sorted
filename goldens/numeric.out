Numeric sorting:
// keep-sorted-test start numeric=yes
BAR_1
BAR_2
BAR_10
BAR_99999999999999999999999999999999999999999999999
FOO_2
FOO_3
FOO_100
// keep-sorted-test end


Mixed numeric and preference sorting with sticky comments:
deployment_state = [
  // keep-sorted-test start numeric=yes prefix_order=INIT,ROLLOUT,COMPLETE
  // Start initialisation
  INIT_1,
  INIT_5,
  INIT_10,
  // Only deploy to 0.1%
  ROLLOUT_0_1,
  // just one percent.
  ROLLOUT_1,
  ROLLOUT_5,
  ROLLOUT_10,
  ROLLOUT_50,
  // Nearly done...
  ROLLOUT_100,
  // All done.
  COMPLETE,
  // keep-sorted-test end
]

Prefix containing a number with numeric sorting:
droid_components = [
  // keep-sorted-test start numeric=yes prefix_order=R2,C3
  R2D2_BOLTS_5_MM,
  R2D2_BOLTS_10_MM,
  R2D2_PROJECTOR,
  C3PO_ARM_L,
  C3PO_ARM_R,
  C3PO_HEAD,
  R4_MOTIVATOR,
  // keep-sorted-test end
]
